fractal_server/app/models/project.py

```diff
@@ -20,25 +21,32 @@ class Project(_ProjectBase, SQLModel, table=True):
         back_populates="project_list",
         sa_relationship_kwargs={
             "lazy": "selectin",
+            "order_by": "UserOAuth.email",
         },
     )
 
     dataset_list: list[Dataset] = Relationship(
         sa_relationship_kwargs={
             "lazy": "selectin",
             "cascade": "all, delete-orphan",
+            "order_by": "Dataset.name",
+            "collection_class": ordering_list("name"),
         }
     )
 
     workflow_list: list[Workflow] = Relationship(
         sa_relationship_kwargs={
             "lazy": "selectin",
             "cascade": "all, delete-orphan",
+            "order_by": "Workflow.name",
+            "collection_class": ordering_list("name"),
         },
     )
 
     job_list: list[ApplyWorkflow] = Relationship(
         sa_relationship_kwargs={
             "lazy": "selectin",
+            "order_by": "ApplyWorkflow.id",
```

Comment:
Let's try with `start_timestamp`, possibly in reverse order (latest first, older last).